kebechet/managers/update/update.py

```diff
@@ -328,22 +339,25 @@ def _replicate_old_environment(cls) -> None:
         cls.run_pipenv('pipenv sync --dev')
 
     @classmethod
-    def _create_pipenv_environment(cls) -> None:
-        """Create a pipenv environment - Pipfile and Pipfile.lock from requirements.in file."""
-        if not os.path.isfile('requirements.in'):
+    def _create_pipenv_environment(cls, input_file: str) -> None:
+        """Create a pipenv environment - Pipfile and Pipfile.lock from requirements.in or requirements-dev.in file."""
+        if os.path.isfile(input_file):
+            _LOGGER.info(f"Installing dependencies from {input_file}")
+            cls.run_pipenv(f'pipenv install -r {input_file}')
```

Comment:
I think there will be needed ``--dev`` flag in case of ``requirements-dev.in`` to propagate "is dev dependency correctly".

Reply:
@harshad16 ping